src/services/IClock.hh

```diff
@@ -9,18 +9,25 @@ class IClock
 {
 public:
     virtual ~IClock() noexcept = default;
+    IClock(const IClock&) noexcept = delete;
+    IClock& operator=(const IClock&) noexcept = delete;
+    IClock(IClock&&) noexcept = delete;
+    IClock& operator=(IClock&&) noexcept = delete;
 
     /// <summary>
     /// Gets the current system time.
     /// </summary>
-    /// <remarks>May change if the user adjusts their clock or the system does (daylight savings).</remarks>
+    /// <returns>May change if the user adjusts their clock or the system does (daylight savings).</returns>
```

Comment:
This does not describe the return value.

Reply:
Put remarks as before.